`DumpEMFields` doesn't compile after include changes

When compiling master, I get:
```text
.../opalx/src/BasicActions/DumpEMFields.cpp: In member function 'void DumpEMFields::writeFieldLine(const std::set<std::shared_ptr<Component> >&, Vector_t<double, 3>&, const double&, std::ofstream&) const':
.../opalx/src/BasicActions/DumpEMFields.cpp:297:47: error: 'Units' has not been declared
  297 |         fout << point[0] << " " << point[1] * Units::rad2deg << " "
      |                                               ^~~~~
make[2]: *** [src/CMakeFiles/opalx.dir/BasicActions/DumpEMFields.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [src/CMakeFiles/opalx.dir/all] Error 2
make: *** [all] Error 2
```
Looks like there is an include problem here (I suppose it has something to do with `OPALTypes` changes that weren't pulled into the `DumpEMFields` branch first).

--- a/src/BasicActions/DumpEMFields.cpp
+++ b/src/BasicActions/DumpEMFields.cpp
@@ -21,6 +21,7 @@
 #include "Attributes/Attributes.h"
 #include "Utilities/OpalException.h"
 #include "Utilities/Util.h"
+#include "Physics/Units.h"
 #include <filesystem>
 #include <fstream>
 

